README.md

```diff
@@ -19,164 +20,141 @@ You should demonstrate an ability to:
 
 ## Wave 1
 
 ### Learning Goals
-- Create a **class** inside of a **module**
-- Create **methods** inside the **class** to perform actions
-- Learn how Ruby does error handling
+- Create a **class**
+- Write **instance methods** inside a **class** to perform actions
+- Link two classes using **composition**
+- Use exceptions to handle errors
 - Verify code correctness by **testing**
 
 ### Testing
 For Wave 1, all tests have been provided for you. For each piece of functionality that you build, you should run the tests from the command line using the `rake` command. To focus on only one test at a time, change all `it` methods to `xit` except for the **one test** you'd like to run. All tests provided should be passing at the end of your work on Wave 1.
 
 ### Requirements
 
-Create a `Grocery` module which will contain an `Order` class and any future grocery store logic.
+#### Customer
+
+Create a class called `Customer`. Each new Customer should include the following attributes:
+- ID, a number
+- Email address, a string
+- Delivery address, a hash
+
+ID should be _readable_ but not _writable_; the other two attributes can be both read and written.
+
+#### Order
 
 Create an `Order` class which should have the following functionality:
 - A new order should be created with:
-  - an ID, read-only
-  - a collection of products and their cost
-    - zero products is permitted
-    - you can assume that there is **only one** of each product
+  - ID, a number (read-only)
+  - A collection of products and their cost
+    - Zero products is permitted
+    - You can assume that there is **only one** of each product
+  - An instance of `Customer`, the person who placed this order
+  - A `fulfillment_status`, a symbol, one of `:pending`, `:paid`, `:processing`, `:shipped`, or `:complete`
+    - If no `fulfillment_status` is provided, it will default to `:pending`
+    - If a status is given that is not one of the above, an `ArgumentError` should be raised
 - A `total` method which will calculate the total cost of the order by:
-  - summing up the products
-  - adding a 7.5% tax
-  - ensure the result is rounded to two decimal places
+  - Summing up the products
+  - Adding a 7.5% tax
+  - Rounding the result to two decimal places
 - An `add_product` method which will take in two parameters, product name and price, and add the data to the product collection
-  - It should return `true` if the item was successfully added and `false` if it was not
-
+  - If a product with the same name has already been added to the order, an `ArgumentError` should be raised
 
 ### Optional:
 Make sure to write tests for any optionals you implement!
 
 - Add a `remove_product` method to the `Order` class which will take in one parameter, a product name, and remove the product from the collection
```

Comment:
Should we tell them what a return value should be?

Reply:
I don't feel like this really needs to return anything.

Reply:
then we should maybe say that it returns the product itself again? idkdkkdjfalskdfj

Reply:
Could we just ask them to consider what the method should return?